src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -189,7 +189,7 @@ public String t3XMP() {
                 + "</rdf:Seq></bibtex:editor>" + "<bibtex:bibtexkey>Clarkson06</bibtex:bibtexkey>"
                 + "<bibtex:journal>International Journal of High Fidelity</bibtex:journal>"
                 + "<bibtex:booktitle>Catch-22</bibtex:booktitle>" + "<bibtex:pdf>YeKis03 - Towards.pdf</bibtex:pdf>"
-                + "<bibtex:keywords>peanut,butter,jelly</bibtex:keywords>"
+                + "<bibtex:keywords>peanut, butter, jelly</bibtex:keywords>"
```

Comment:
The problem is that one of the test simply copies the keywords field as is and one parses it resulting in two differently looking fields at the end.